scripts/make_docs.py

```diff
@@ -59,16 +61,30 @@
 
 ## Available selectors
 
-{%- for key, val in platforms|dictsort %}
+{%- for key, val in selectors|dictsort %}
 - `{{key}}`
-{% endfor %}
+{%- endfor %}
+
+## Available Platforms
+Specify which platform to build for via the `--platform` argument. If provided, this argument must be formated as `<platform>-<architecture>`
+{%- for platform in supported_platforms %}
+- `{{platform}}`
+{%- endfor %}
+
+The following options are valid but not actively tested.
+{%- for platform in unsupported_platforms %}
+- `{{platform}}`
+{%- endfor %}
```

Comment:
This is the same list as listed before... please just remove it... there will be always more platforms supported by conda but maybe not yet used/tested by constructor while maybe still working. Trying to list those will always result in an outdated list and an unclear statement.
```suggestion
```

Reply:
sorry, my comment was not correct. updated.

Reply:
[3001ae9](https://github.com/conda/constructor/pull/620/commits/3001ae9ee472c521ec6abf2596af2926a33489cd)
removed the mention of untested, but technically valid, platforms from the documentation

Reply:
[e8bdc9e](https://github.com/conda/constructor/pull/620/commits/e8bdc9ec60f4dc91bd83c0ae082f733d37fcaa2c)
I think the "duplicate entries" comment was referring to `linux-arch64` and `linux-aarch64`. This was a typo.

Reply:
dd54cfd
No need to ask conda what it supports